xbmc/addons/AddonManager.cpp

```diff
@@ -279,6 +287,14 @@ static bool LoadManifest(std::set<std::string>& system, std::set<std::string>& o
   return true;
 }
 
+CAddonMgr::CAddonMgr()
+  : m_apiVersions(API_VERSIONS.begin(), API_VERSIONS.end())
+{ }
+
+CAddonMgr::CAddonMgr(std::vector<APIVersion> apiVersions)
```

Comment:
Why you have added the class constructor with value? It should only be one time created by CServiceManager.